addons/mail/models/res_company.py

```diff
@@ -38,3 +45,21 @@ def _compute_email_formatted(self):
                 company.email_formatted = company.catchall_formatted
             else:
                 company.email_formatted = ''
+
+    @api.depends('primary_color')
+    def _compute_email_primary_color(self):
+        """ When updating documents layout colors, force usage of same colors
+        for emails as it is considered as base colors for all communication.
+        Inverse is not true, people may change email colors without changing
+        their overall layout. """
+        for company in self:
+            company.email_primary_color = company.primary_color or '#000000'
```

Comment:
Note that adding fallbacks everywhere makes it difficult to update, not sure if we should consider filling up that field when "primary_color" is unset on company.

Reply:
Not sure to understand. I prefer to have settings displaying a default value, and be defensive in emails / layouts in addition ...